Dockerfile

```diff
@@ -68,8 +81,11 @@ COPY --from=quay.io/terraform-docs/terraform-docs:latest /usr/local/bin/terrafor
 COPY --from=tflint-build /usr/local/bin/tflint /usr/local/bin/tflint
 COPY --from=alzlibtool-build /root/go/bin/alzlibtool /usr/local/bin/alzlibtool
 COPY --from=pwsh-build /opt/microsoft/powershell/7 /opt/microsoft/powershell/7
+COPY --from=bicep-langserver-build /opt/bicep-langserver /opt/bicep-langserver
 
-RUN ln -s /opt/microsoft/powershell/7/pwsh /usr/bin/pwsh \
+RUN echo "#!/bin/bash\n\nexec dotnet /opt/bicep-langserver/Bicep.LangServer.dll" > /usr/local/bin/bicep-langserver \
```

Comment:
Only a nitpick: I'd opt to name the script bicep-lsp as this seems to be the common name of the script used in other deployments.